source/Irrlicht/FW_Mutex.h

```diff
@@ -198,7 +224,53 @@ not using add because we atomically wait for value to be 0 before swapping
 **/
 #define FW_AtomicCounterMagicBlockVal 0x1000000
 
-#if _MSC_VER && !__INTEL_COMPILER
+#if defined(FW_MUTEX_H_CXX11_IMPL)
+
+#define FW_AtomicCounter std::atomic_int
```

Comment:
std::atomic is neither copyable nor movable. -- volatile int used to work for this...

but we never copy or move these